mailpoet/lib/Services/AuthorizedEmailsController.php

```diff
@@ -191,4 +212,11 @@ private function validateAuthorizedEmail($authorizedEmails = [], $email = '') {
     $lowercaseAuthorizedEmails = array_map('strtolower', $authorizedEmails);
     return in_array(strtolower($email), $lowercaseAuthorizedEmails, true);
   }
+
+  private function validateEmailDomainIsVerified($verifiedDomains = [], $email = ''): bool {
```

Comment:
Just a detail, but could you please add type hints? We want to use type hints in the new code.

`validateEmailDomainIsVerified(array $verifiedDomains = [], string $email = '')`

Reply:
Fixed in https://github.com/mailpoet/mailpoet/pull/4363/commits/236b75eddf091510f9beaf12e3923629cced2ace